### PR TITLE
Refactor padding styles

### PR DESCRIPTION
**Pull request by shanberg**
Paddings and margins are now determined more clearly on a scale. This should make it easier to read (and compute syntax of) longer spacing declarations, as well as make it more intuitive to stick to a regular spacing interval.

### CODE REVIEW

File: src/style/functions.scss [hunk 1]
```diff
@@ -133,3 +133,8 @@
     @return $colors;
   }
 }
+
+// Scales
+@function spacing-scale($int) {
```
Comment (spmcbride1201) [inline src/style/functions.scss]: Can you rename the variable from `$int` to `$factor` or something? When I see integers, I think "whole numbers without decimal points," but we clearly want to be able to but but any arbitrary value in if it's an integer or not.

**Comment by shanberg** [comment]
@spmcbride1201 Fixed.

**Comment by shanberg** [comment]
@spmcbride1201 My rebase got a little hairy. Would you mind taking another look before I merge?

**Comment by spmcbride1201** [comment]
@shanberg - Of course I'll take a look. No worries.

**Comment by spmcbride1201** [comment]
@shanberg - It seems to work for me, and the source files look sane. Feel free to squash - merge.

**Pull request merged**